plugins/core/core_misc.py

```diff
@@ -94,20 +94,29 @@ def onjoin(conn, bot):
     if oper_pw and oper_user:
         out = "oper {} {}".format(oper_user, oper_pw)
```

Comment:
```suggestion
        out = "OPER {} {}".format(oper_user, oper_pw)
```
IRC commands are always uppercase.

Reply:
Why did you remove a space?